docs/api-reference/composite-layer.md

```diff
@@ -78,9 +78,10 @@ functions called.
 The default implementation returns `pickParams.info` without any change.
 
 
-##### `getBaseLayerProps`
+##### `getSubLayerProps`
 
-Returns an object with the key base `Layer` props that `CompositeLayer`s typically forward to their sublayers.
```

Comment:
We should mention the special handling of id and updateTriggers. This is new compared to the old function.

This function now also creates the complete sublayer props object, not just the core props.